app/page/home/index.tsx

```diff
@@ -36,16 +48,58 @@ export function Home() {
 }
 
 function AccountCreation() {
-  const [, navigate] = useHashLocation()
+  const { provider } = useContext(ProviderContext)
+  const { createSimpleAccount } = useAction()
+  const network = useNetwork()
+
+  useEffect(() => {
+    const initSimpleAccount = async () => {
+      const hasSimpleAccountFactory = address.isValid(
+        network.accountFactory[AccountType.SimpleAccount]
+      )
+
+      if (hasSimpleAccountFactory) {
+        const account = await SimpleAccount.initWithFactory(provider, {
+          ownerPrivateKey: Wallet.createRandom().privateKey,
+          salt: number.random(),
+          factoryAddress: network.accountFactory[AccountType.SimpleAccount]
+        })
+
+        await createSimpleAccount(`Account 1`, account, network.id)
+      }
+    }
+
+    initSimpleAccount()
+  }, [network.id])
 
   return (
-    <div className="text-center">
+    <div className="flex items-center justify-between mb-[16px] mt-[4px]">
+      <div>
+        <NullAccountSelector />
+      </div>
+      <div>
+        <NetworkSelector />
+      </div>
+    </div>
```

Comment:
It looks like <Navbar />, just different in the sub-component `<NullAccountSelector />` 🧐 
I think the previous one looks fine to me.